components/appointmentNotesArea/appointmentNotesArea.php

```diff
@@ -18,14 +18,14 @@
 	<h4>Add a Note</h4>
 	<textarea ng-model="noteToAddText" 
 		placeholder="-- Expanation --" 
-		class="form-control note-textarea" 
+		class="form-control note-textarea dcf-input-text" 
 		cols="300" 
 		rows="3" 
 		maxlength="{{MAX_NOTE_LENGTH}}" 
 		ng-maxlength="MAX_NOTE_LENGTH">
 	</textarea>
-	<span class="wdn-pull-right margin-top">{{ noteToAddText ? noteToAddText.length : 0 }}/{{MAX_NOTE_LENGTH}}</span>
-	<button class="wdn-button wdn-button-triad margin-top" 
+	<span class="dcf-float-right dcf-mt-1">{{ noteToAddText ? noteToAddText.length : 0 }}/{{MAX_NOTE_LENGTH}}</span>
+	<button class="dcf-btn dcf-btn-primary dcf-mt-3" 
```

Comment:
So I've noticed that some of the buttons use `class="dcf-btn"` and some use `class="dcf-btn dcf-btn-primary"`. Should we be specifying primary on ones that just have `dcf-btn`?

Reply:
I looked into this a little more and having the primary part will provide the hovering effect to the buttons, so I will go ahead and update the buttons that are missing it.